docs/guides/upgrades/upgrade_node.md

```diff
@@ -197,6 +201,21 @@ You can check the status with:
 systemctl status evmosd
 ```
 
+#### Update Cosmosvisor to V2
```

Comment:
should probably make it clear somewhere that people should be on `v1.1.2` prior to this upgrade, or it won't work